v3/integrations/nrmongo/go.mod

```
@@ -2,10 +2,11 @@ module github.com/newrelic/go-agent/v3/integrations/nrmongo
 
 // As of Dec 2019, 1.10 is the mongo-driver requirement:
 // https://github.com/mongodb/mongo-go-driver#requirements
-go 1.17
+go 1.18
 
 require (
-	github.com/newrelic/go-agent/v3 v3.18.2
+	github.com/newrelic/go-agent/v3 v3.22.0
```

```suggestion
	github.com/newrelic/go-agent/v3 v3.22.1
```

We probably should not require the version of the agent we recently dropped from the go eco system.